src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/prison/PrisonEventListenerIntTest.kt

```diff
@@ -70,8 +72,10 @@ class PrisonEventListenerIntTest : MessagingMultiNodeTestBase() {
     val nationalInsuranceNumber = randomNationalInsuranceNumber()
     val driverLicenseNumber = randomDriverLicenseNumber()
     val ethnicity = randomEthnicity()
+    val sentenceStartDate = randomDate()
+    val primarySentence = Random.nextBoolean()
```

Comment:
Maybe just keep this as a const true / false as this impacts the return statement